src/waterfall-cli.ts

```diff
@@ -147,26 +172,34 @@ module.exports = function Constructor(customSettings) {
 	}
 
 	// Form execution paths
-	const executionPaths = [];
+	const executionPaths: string[] = [];
 	let currentPathPrefix = path.dirname(settings.mainFilename);
-	const commandPieces = organizedArguments.command.trim().split(' ');
+	const commandPieces: string[] = organizedArguments.command.trim().split(' ');
 
 	commandPieces.forEach((command, index) => {
 		// Update current path prefix
 		currentPathPrefix = path.join(currentPathPrefix, command);
 
 		// Get the files we care about
-		const commandFiles = utils(settings).files.getFiles(currentPathPrefix);
+		const commandFiles: string[] = utils(settings).files.getFiles(
+			currentPathPrefix
+		);
 
 		// Get the command path
 		const commandPath = commandFiles.filter(path => path.match(/\.js$/))[0];
 
 		// Get spec
 		const specFilePath = commandFiles.filter(path => path.match(/\.json$/))[0];
-		let spec = {};
+		let spec: CommandSpec = {
```

Comment:
After you make the data's `description` optional, along with the `flags` and `options` keys, you should be able to leave this as `let spec = {};`.